src/main/java/io/wcm/caconfig/extensions/references/impl/ConfigurationReferenceProvider.java

```diff
@@ -200,4 +210,54 @@ private static String getType() {
     return REFERENCE_TYPE;
   }
 
+  /**
+   * Adds asset references from the given page to the provided list of asset references.
+   * This method checks the properties of the `jcr:content` node of the modified page for any
+   * references to assets in the `/content/dam/` path. If such references are found, they are
+   * added to the list of asset references.
+   *
+   * @param assetReferences The list to which asset references will be added.
+   * @param configPage The page whose properties are checked for asset references.
+   * @param resourceResolver The resource resolver used to resolve asset paths.
+   */
+  private void AddAssetReferencesOfaResource(List<com.day.cq.wcm.api.reference.Reference> assetReferences, Page configPage, ResourceResolver resourceResolver) {
+    Resource configPageContentRes = configPage.getContentResource();
+    if (configPageContentRes == null) {
+      return;
+    }
+    ValueMap properties = configPageContentRes.getValueMap();
+    for (String propertyName : properties.keySet()) {
+      Object propertyValue = properties.get(propertyName);
+      if (propertyValue instanceof String && ((String) propertyValue).startsWith(DamConstants.MOUNTPOINT_ASSETS)) {
+        Resource assetResource = resourceResolver.getResource((String) propertyValue);
+        if (isAssetReference(assetResource)) {
+          assetReferences.add(new com.day.cq.wcm.api.reference.Reference(DamConstants.ACTIVITY_TYPE_ASSET, assetResource.getName(), assetResource, getAssetLastModified(assetResource)));
+        }
+      }
+    }
+  }
+
+
+  private static long getAssetLastModified(Resource assetResource) {
```

Comment:
the `com.day.cq.dam.api.Asset` interface has a getLastModified() method, please use that.